test/app-profile.js

```diff
@@ -155,7 +155,7 @@ describe('Bigtable/AppProfile', function() {
         options_,
         callback
       ) {
-        assert.strictEqual(name, appProfile.name);
+        assert.strictEqual(name, appProfile.id);
```

Comment:
please change the `name` field to be `id`

Reply:
@sduskis  :Changes committed as per comments.